src/Compilers/Core/Portable/SourceGeneration/GeneratorDriverState.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+
+#nullable enable
+namespace Microsoft.CodeAnalysis
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0016:Add public types and members to the declared API", Justification = "<Pending>")]
+    public readonly struct GeneratorDriverState
+    {
+        internal GeneratorDriverState(Compilation compilation, ParseOptions parseOptions)
+            : this(compilation, parseOptions, ImmutableArray<GeneratorProvider>.Empty, ImmutableArray<AdditionalText>.Empty, ImmutableArray<PendingEdit>.Empty, ImmutableDictionary<GeneratorProvider, ImmutableArray<GeneratedSourceText>>.Empty, null, false)
+        {
+        }
+
+        internal GeneratorDriverState(Compilation compilation, ParseOptions parseOptions, ImmutableArray<GeneratorProvider> providers, ImmutableArray<AdditionalText> additionalTexts, ImmutableArray<PendingEdit> edits, ImmutableDictionary<GeneratorProvider, ImmutableArray<GeneratedSourceText>> sources, Compilation? finalCompilation, bool editsFailed)
```

Comment:
>(Compilation compilation, ParseOptions parseOptions, ImmutableArray<GeneratorProvider> providers, ImmutableArray<AdditionalText> additionalTexts, ImmutableArray<PendingEdit> edits, ImmutableDictionary<GeneratorProvider, ImmutableArray<GeneratedSourceText>> sources, Compilation? finalCompilation, bool editsFailed) [](start = 37, length = 314)

Please split this line into multiple lines so that all parameters were visible